src/routes/+page.ts

```diff
@@ -38,7 +42,7 @@ async function requestUndecidedPapers(project: Project): Promise<PaperListEntryI
  * - open reviews from the project
  */
 export const load: PageLoad = async () => {
-    const thisUserId = "0";
+    const thisUserId = (await backendService.getCurrentUser(Nothing).response).id;
     const allUserProjects = backendService.getAllProjectsForUser({ id: thisUserId }).response;
```

Comment:
Make this asynchronous to not block the page loading.
```suggestion
    const allUserProjects = backendService
        .getCurrentUser(Nothing)
        .response.then((user) => backendService.getAllProjectsForUser({ id: user.id }).response);
```

Reply:
And add the default catch after it, so the page does not crash ... oh oh oh, I really was not concentrated.